hg-fast-export.py

```diff
@@ -336,7 +336,7 @@ def get_branchname(name):
     if fn_encoding:
       filename=filename.decode(fn_encoding).encode('utf8')
     filename=strip_leading_slash(filename)
-    if filename=='.hgsubstate':
+    if filename=='.hgsubstate' or filename=='.hgsub':
```

Comment:
Use `context.substate` to check subrepos

Smth like:
```suggestion
    if (filename=='.hgsubstate' or filename=='.hgsub') and not context.substate:
```